docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 
 # Basic Components
 
-Nodes work with a data records. They can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
+Nodes work with data records. They can produce, fetch, send and collect data or organize the flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
 
-Most of the nodes, with source and sink nodes being notable exceptions, have both input and at least one output flow.
+Most of the nodes have both input and at least one output flow. Source and sink nodes are an exception.
 
 Sinks and filters can be disabled by selecting `Disable` checkbox.
 
 &nbsp;
 ## Variable
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable holds its content and  its value is assigned to a newly declared `myFirstVariable` variable.
+A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the event has been read from a Kafka topic, the `#input` variable holds event contents. Event value is assigned to a newly declared `myFirstVariable` variable.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable0.png   "Scenario with variable declaration")
 
 
-As you can see in the `variable` configuration form below, Nussknacker inferred the data type of the `#input` variable from the information already available to Nussknacker.
+As you can see in the `variable` configuration form below, Nussknacker has inferred the data type of the `#input` variable. Nussknacker can do this based on info stored in the previous components.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable1.png "Variable declaration form")
 
 
-In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain `operation` field, the value of this field can be obtained in the following way:
+In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain an `operation`, the value can be obtained using the following pattern:
```

Comment:
maybe "an 'operation' field" would be better, as it explicitly states what is this 'operation' entity

Reply:
I'm not sure whether this specification is needed for a non-technical user. How would he know what "field" means here? I think it makes things more complicated. I could probably settle for 'operation' value which is technically incorrect, but probably more understandable to most people.

Reply:
I think that if someone knows what JSON is that person may also know what json object field is. In that case "they contain an `operation`" may mean to them less then "they contain an `operation` field". But it is more of an sugggestion, it can stay like this.